README.md

```diff
@@ -34,23 +28,22 @@
 └── aksel           # @navikt/aksel, kommandolinje-verktøy
 
 aksel.nav.no/
-├── website # NEXT.JS app (v13)
-├── website/sanity # Sanity CMS-app (v3)
-└── playroom # aksel.nav.no/sandbox
-
+├── website         # Next.js-app
+├── website/sanity  # Sanity CMS-app
```

Comment:
I removed the version numbers since we probably will forget to update them anyways, and I didn't really see any need to include it.